Commands/LASettingsCommand.cs

```diff
@@ -0,0 +1,13 @@
+using LocalAdmin.V2.IO;
+using LocalAdmin.V2.Commands.Meta;
+using LocalAdmin.V2.Core;
+using System;
+
+namespace LocalAdmin.V2.Commands;
+
+internal sealed class LASettingsCommand : CommandBase
+{
+    Config Settings;
+    public LASettingsCommand() : base("settings") { }
```

Comment:
You might want to rename the command itself, not the class.

As I explained to @Jesus-QC here https://github.com/northwood-studios/LocalAdmin-V2/pull/72#issuecomment-1676370734 having such command prevents executing "settings" command on the server console.

A similar command is named "lacfg". You might want to rename this to "lasettings" or actually merge these two commands into one, so "lacfg" prints both the path and the configuration.

Reply:
OK, I'll merge lacfg with settings and make it the main way to see configuration settings.